Examples/Swift/ManageOfflineRegionsExample.swift

```diff
@@ -111,23 +94,28 @@ class ManageOfflineRegionsExample_Swift: UIViewController, MGLMapViewDelegate, U
         }
     }
 
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        let label = UILabel()
+        if section == 0 {
+            label.backgroundColor = UIColor.white
+            label.textAlignment = .center
+            label.text = "No Offline Packs Saved"
+            return label.text
+        }
+
+        return nil
+    }
+
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        var cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+
         cell = UITableViewCell(style: .subtitle, reuseIdentifier: "cell")
```

Comment:
Ohhh see what you mean now with the cell style thing. Here's a simply approach I think could work:

```swift
let cell = tableView.dequeueReusableCell(withIdentifier: "cell") ?? UITableViewCell(style: .subtitle, 
                                                                    reuseIdentifier: "cell")
```

This checks `dequeueReusableCell` to see if there is already a cell prepared with that identifier, and if there isn't, it creates a new cell with the `subtitle` style.

Reply:
changing the code to this still doesn't allow the subtitle to show :(